src/org/zaproxy/zap/extension/autoupdate/AutoUpdateAPI.java

```diff
@@ -17,32 +17,55 @@
  */
 package org.zaproxy.zap.extension.autoupdate;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Vector;
+
 import net.sf.json.JSONObject;
 
+import org.apache.commons.lang.ObjectUtils;
 import org.apache.log4j.Logger;
+import org.zaproxy.zap.Version;
+import org.zaproxy.zap.control.AddOn;
 import org.zaproxy.zap.extension.api.ApiAction;
 import org.zaproxy.zap.extension.api.ApiException;
+import org.zaproxy.zap.extension.api.ApiException.Type;
 import org.zaproxy.zap.extension.api.ApiImplementor;
 import org.zaproxy.zap.extension.api.ApiResponse;
 import org.zaproxy.zap.extension.api.ApiResponseElement;
+import org.zaproxy.zap.extension.api.ApiResponseList;
+import org.zaproxy.zap.extension.api.ApiResponseSet;
 import org.zaproxy.zap.extension.api.ApiView;
 
 public class AutoUpdateAPI extends ApiImplementor {
 
     private static Logger log = Logger.getLogger(AutoUpdateAPI.class);
 
 	private static final String PREFIX = "autoupdate";
-	private static final String DOWNLOAD_LATEST_RELEASE = "downloadLatestRelease";
+	private static final String ACTION_DOWNLOAD_LATEST_RELEASE = "downloadLatestRelease";
+    private static final String ACTION_INSTALL_ADDON = "installAddon";
```

Comment:
Could use the same indentation as the surrounding statements (same for other statements).